classes/sdimg.bbclass

```diff
@@ -33,116 +34,65 @@ SDIMG_PARTITION_ALIGNMENT_MB ?= "8"
 
 ########## CONFIGURATION END ##########
 
+inherit image_types
 
-# This script depends on:
-#     util-linux (fdisk)
-#     dosfstools (mkfs.vfat)
-#     mtools     (mcopy)
-#     e2fsprogs  (resize2fs)
-IMAGE_DEPENDS_sdimg = "util-linux-native dosfstools-native mtools-native e2fsprogs-native"
+WKS_FULL_PATH = "${WORKDIR}/mender-sdimg.wks"
 
 # We need to have the ext3 image generated already
 IMAGE_TYPEDEP_sdimg = "ext3"
 
+IMAGE_DEPENDS_sdimg = "${IMAGE_DEPENDS_wic}"
```

Comment:
This is defined inside `image_types.bbclass`, and we are using it here to make sure we depend on all the same things, since we will use wic for the job. If you see a bit further up you'll see the `inherit image_types`.